sw/device/examples/hello_world/BUILD

```diff
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 load("//rules/opentitan:defs.bzl", "opentitan_binary")
-load("//rules:exclude_files.bzl", "exclude_files")
+load("//rules:files.bzl", "exclude_files")
```

Comment:
nit: for a follow up PR if applicable. Should we remove this load line? `exclude_files` is no longer used in this file.

Reply:
yes.